src/PLS.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27929.0
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29003.217
```

Comment:
Are we switching to VS19?

Reply:
Everyone is on it AFAIK... I am on dogfood, works OK, some stuff is better/faster.

Reply:
Ok, I was still on VS15